cmd/admin-rpc-client.go

```diff
@@ -129,12 +145,92 @@ func (lc localAdminClient) GetConfig() ([]byte, error) {
 func (rc remoteAdminClient) GetConfig() ([]byte, error) {
 	args := AuthRPCArgs{}
 	reply := ConfigReply{}
-	if err := rc.Call("Admin.GetConfig", &args, &reply); err != nil {
+	if err := rc.Call(getConfigRPC, &args, &reply); err != nil {
 		return nil, err
 	}
 	return reply.Config, nil
 }
 
+// WriteTmpConfig - writes config file content to a temporary file on
+// the local server.
+func (lc localAdminClient) WriteTmpConfig(tmpFileName string, configBytes []byte) (int64, error) {
+	configDir, err := getConfigPath()
+	if err != nil {
+		errorIf(err, "Failed to get config path")
+	}
+
+	tmpFile, err := os.Create(filepath.Join(configDir, tmpFileName))
+	if err != nil {
+		errorIf(err, "Failed to create temporary config file.")
+		return 0, err
+	}
+
+	defer tmpFile.Close()
+
+	config := bytes.NewReader(configBytes)
```

Comment:
May be just use `ioutil.WriteFile`  avoids all the clutter of `os.Create, io.Copy and bytes.NewReader`